app/routes/web.php

```diff
@@ -67,3 +67,4 @@
 Route::get('api/locations', function (Request $request) {
 	return Location::all();
 });
+Route::post('saveuserlocation', 'HomeController@saveuserlocation');
```

Comment:
Along with renaming your HomeController method, you'll need to rename it in your route in web.php.

For the route name, separate the words with hyphens to be consistent with other routes.  Rename the route to save-user-location in web.php and your ajax calls from home.js.